draft-ietf-quic-multipath.md

```diff
@@ -535,6 +481,11 @@ The Path ID 0 indicates the initial path of the connection.
 Respectively, the connection IDs used during the handshake belong to the initial path
 with Path ID 0.
 
+Endpoints SHOULD also use
+the PATH_NEW_CONNECTION_ID frame to provide new connection IDs for Path ID 0 and,
+respectively, the PATH_RETIRE_CONNECTION_ID frame to retire connection IDs for
+Path ID 0.
```

Comment:
I wonder if such text should not be part of Section 2.3 as well, as the Path ID 0 link between ACK and PATH_ACK is the same than the NEW_CONNECTION_ID/PATH_NEW_CONNECTION_ID and RETIRE_CONNECTION_ID/PATH_RETIRE_CONNECTION_ID. I think such clarification should be stated once and would be better suited in its own subsection.

Reply:
Sorry I don't know which section you mean by 2.3? I think this PR together with PR #507 will then have both things together in the same section. Or where do you think this is stated again?

Reply:
2.3 section was referring to the section related to "Handling ACK and PATH_ACK in 0-RTT and 1-RTT", where it discusses relation between ACK and PATH_ACK. This text could be extended to the NEW/RETIRE_CID frames. But this is not blocking and we can merge now.